packages/builder/src/build.js

```diff
@@ -66,7 +71,10 @@ if (!skpmConfig.manifest) {
 }
 
 const output = path.join(process.cwd(), skpmConfig.main)
-const manifest = path.join(process.cwd(), skpmConfig.manifest)
+const manifest = path.join(
+  process.cwd(),
+  argv.manifest ? argv.manifest : skpmConfig.manifest
```

Comment:
if you want to go the way of an argument instead of an env variable, I'd rather pass `argv` to `getSkpmConfigFromPakageJSON` and the override there